File: AnDsSistema/2024 Semestre 1/2a-TecMov/Aula02/tde_media_cores.kt
fun getAvg(values : MutableList<Int>):Int{
    if(values.isEmpty()){
      print("Lista vazia");
      return 0;
    }else if(values == null){
      print("Lista... nula?");
      return 0;
    }
    var totalSum : Int = 0;
    for(i in values){
      totalSum += i;
    }
    return totalSum / values.size;
  }

fun translateColor(color:String):String{
  when(color.lowercase()){
    "red"->return "Vermelho";
    "blue"->return "Azul";
    "green"->return "Verde";
    "yellow"->return "Amarelo";
    else->return "Qualquer outra cor.";
  }
}

fun main() {
  var entryValues : MutableList<Int> = mutableListOf (1,573,9,32,2);
  //List<int> entryValues = [];
  //var entryValues = null;
  println(getAvg(entryValues));   
  //Coincidentemente, o resultado é 123.4, ou por ser Int, 123

  println(translateColor("yelLOw"));
}

File: AnDsSistema/2024 Semestre 1/5a-SisMov/Aula02/tde_folder/tde_media_cores/bin/tde_media_cores.dart
double getAvg(List<int> values){
    if(values.isEmpty){
      print("Lista vazia");
      return 0.0;
    }else if(values == Null){
      print("Lista... nula?");
      return 0.0;
    }
    int totalSum = 0;
    values.forEach((element)=>totalSum+=element);
    return totalSum / values.length;
  }

String translateColor(String color){
  switch(color.toLowerCase()){
    case "red":
      return "Vermelho";
    case "blue":
      return "Azul";
    case "green":
      return "Verde";
    case "yellow":
      return "Amarelo";
    default:
      return "Qualquer outra cor.";
  }
}

void main() {
  List<int> entryValues = [1,573,9,32,2];
  //List<int> entryValues = [];
  //var entryValues = null;
  print(getAvg(entryValues));   
  //Coincidentemente, o resultado é 123.4

  print(translateColor("yelLOw"));
}
